ApplicationLibCode/ProjectDataModel/Summary/RimSummaryPlotNameHelper.cpp

```diff
@@ -102,8 +102,14 @@ QString RimSummaryPlotNameHelper::plotTitle() const
 //--------------------------------------------------------------------------------------------------
 ///
 //--------------------------------------------------------------------------------------------------
-bool RimSummaryPlotNameHelper::isPlotDisplayingSingleVectorName() const
+bool RimSummaryPlotNameHelper::isPlotDisplayingSingleCurve() const
 {
+    if ( m_analyzer->quantities().size() == 2 && m_analyzer->onlyCrossPlotCurves() )
```

Comment:
Condiftion `m_analyzer->quantities().size() == 2` is repeated. Merge into one?

Reply:
Fixed